app/controllers/concerns/curation_concerns/collections_controller_behavior.rb

```diff
@@ -13,6 +13,9 @@ module CollectionsControllerBehavior
 
       # This is needed as of BL 3.7
       copy_blacklight_config_from(::CatalogController)
+      configure_blacklight do |config|
+        config.search_builder_class = CurationConcerns::CollectionSearchBuilder
```

Comment:
I'm not sure what you mean.  `selects_collections.rb` already does the same thing.  `file_sets_controller_behavior.rb` and `curation_concern_controller.rb` both have methods for `search_builder_class`.  You're saying none of these do anything?